config/environments/production.rb

```diff
@@ -72,4 +72,11 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # SQL server table config
+  #
+  # In production all out tables exist in a 'schema' called `complete` so we
+  # tell Rails to look for it's internal tables there.
+  config.active_record.schema_migrations_table_name = "complete.schema_migrations"
```

Comment:
Is this also the schema name on the service proper? Do we need to park this in an ENV variable?

Reply:
Yes, it implies the schema name in production and staging will be `complete`, take the point - an ENV variable would document this a little clearer 👍